<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsAddWFSSensorLayerDialogBase</class>
 <widget class="QDialog" name="QgsAddWFSSensorLayerDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add WFS sensor layer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="mWFSLayerLabel">
       <property name="text">
        <string>WFS layer</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="mLayerComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mIdAttributeLabel">
       <property name="text">
        <string>ID attribute</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="mIdAttributeComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mObservableAttributeLabel">
       <property name="text">
        <string>Observable attribute</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="mObservableAttributeComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="mBeginAttributeLabel">
       <property name="text">
        <string>Begin attribute</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="mBeginAttributeComboBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="mEndAttributeLabel">
       <property name="text">
        <string>End attribute</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="mEndAttributeComboBox"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsAddWFSSensorLayerDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsAddWFSSensorLayerDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
